neo4j-admin/backup/aws/operations.go

```diff
@@ -6,18 +6,28 @@ import (
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/feature/s3/manager"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
+	smithyendpoints "github.com/aws/smithy-go/endpoints"
 	"github.com/neo4j/helm-charts/neo4j-admin/backup/common"
 	"log"
 	"os"
 	"strings"
 )
 
+type resolverV2 struct{}
+
+func (*resolverV2) ResolveEndpoint(ctx context.Context, params s3.EndpointParameters) (smithyendpoints.Endpoint, error) {
+	// s3.Options.BaseEndpoint is accessible here:
```

Comment:
Are these intentionally comented out to be used only for debugging or this is a typo?

Reply:
yeah it was just a comment..removed it